app/cafe.py

```diff
@@ -0,0 +1,26 @@
+from datetime import date
+
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        issues_found = 0
+        if "vaccine" not in visitor:
+            issues_found += 1
+            raise NotVaccinatedError
```

Comment:
The `NotVaccinatedError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions, e.g., `raise NotVaccinatedError('Visitor is not vaccinated.')`.